frontend/src/component/feature/FeatureView/FeatureOverview/ReleasePlan/ReleasePlan.tsx

```diff
@@ -379,6 +384,9 @@ export const ReleasePlan = ({
                 )}
             </StyledHeader>
             <StyledBody>
+                <StyledBodyHeader>
```

Comment:
I think we already have useUiFlag hook in this component so we can wrap the whole thing inside